### DISCUSSION TITLE
Welcome to opensafety-mbse Discussions!

### DISCUSSION THREAD

**Discussion by modelint** [ToyotaResearchInstitute/opensafety-mbse#1] [Announcements]
## 👋 Welcome to the MBSE Open Safety Discussion Group
Feel free to post questions about our models or modeling approach and we will do our best to respond.

If you know of any resources or standards we should be referencing, feel free to post links. And if you are interested in collaborating on some level, let us know.

**Comment by ralph-voyager** [comment]
In reference to your medium dot com article, dated March 1, 2022 I replied:
But that's the whole point: driverless = dependent on the type of autonomous vehicle used! AV developers can’t control traffic, the weather, nor legislation. What they CAN control, is the vehicle itself. Wouldn’t you say that: -A- the smaller the vehicle, the more margin there is to maneuver, evade other road users? -B- a vehicle can be formatted in such a way that it has more built-in safety for all involved to begin with? Yes. _More in the comments section under your article._

Ergo: modelling should be related to the real deal. I know for instance that the NHTSA is seriously concerned about the phase between simulations and real-world testing. Better anticipate!

**Reply by modelint** [reply]
Thank for the insightful comment Ralph. There are certainly numerous parameters that are platform specific. No doubt about that.

Nonetheless, our models are oriented toward those aspects that do not vary among platforms. Consider an unprotected inside (left in the USA) turn through a signalized four way intersection. Regardless of a vehicle's size or performance characteristics, it is necessary to yield to oncoming priority traffic. If there are pedestrians in the exit crosswalk (leaving the intersection) with a green light and distant oncoming traffic, it is not advisable to proceed past the encroach position as the oncoming traffic may be blocked while pedestrians continue to cross. But, if the light is transitioning to red, it is probably a reasonable move.

When making a lane change the need to signal in advance and check for open space in the target lane also has nothing to do with a platform's specific performance or size.

Now, a high performance vehicle may well see lane change opportunities unavailable to a delivery van or a truck. But we can still characterize these parameters (vehicle length, max acceleration, etc.) and associate them with a vehicle class as, say, a performance profile, and incorporate that into our models. So, while the models themselves may be platform independent, the data populated into them for a particular vehicle need not be.  / Leon

**Reply by ralph-voyager** [reply]
Thank you for getting back to me. I agree: you and I are talking different things. Not mutually exclusive. Modelling is the excercise, the discipline that should be undertaken and assessed as such, given the way vehicles move, interact, need to proceed by traffic rules etc.

Still, my angle is that mobility is first of all about displacing a transport mode of some sort and size from A to B. I have a logistics background. IMO, when technology is supposed to take over control, the car basically becomes the 'app', the application. Why not reformat the car first to optimize the use of autonomous vehicle technology.

I go even further: 'greening' the car (needing less kWh is HOT because of the Ukraine war) and making it driverless are two sides to the same coin. They can be each others stimulus to implement both! More: "[driverless that works](https://www.docdroid.net/FZHBiGB/driverless-that-works-and-that-is-safe-pdf)"

Btw, [Toyota Ventures Climate Fund](https://medium.com/toyota-ventures/toyota-ventures-climate-fund-bridging-the-gap-between-headwinds-and-innovation-5e0c2697191c) recently asked people to send in their ideas. I let them know that this is out there.
Perhaps worthwhile shortcutting... Cheers, Ralph

**Comment by ralph-voyager** [comment]
Wrote a special outline on the autonomous suitability of sleek-footprint vehicles: [(https://rebrand.ly/5648mfh)]
For a quick scan, check out pages 4-5-6-7-8. If you cannot open the link, I will be more than happy to send the pdf.
